### DISCUSSION TITLE
Elaboration on the term 'PROW'?

### DISCUSSION THREAD

**Discussion by StefanVrecicTP** [openmobilityfoundation/mobility-data-specification#625] [General]
e.g., **"The administrative agency (ie, DOT) drops a device into the PROW using an admin code or similar"**

Can I please get elaboration on this term PROW?

**Comment by thekaveman** [comment]
**P**ublic **R**ight **O**f **W**ay - basically the streets, sidewalks, alleyways, and other public-access "lanes" for travel through a city/county/etc.

See https://en.wikipedia.org/wiki/Right_of_way and https://en.wikipedia.org/wiki/Right-of-way_(transportation) for more.

**Reply by schnuerle** [reply]
We also have a definitions section in MDS here: https://github.com/openmobilityfoundation/mobility-data-specification/blob/main/general-information.md#definitions
